pyccl/__init__.py

```diff
@@ -24,6 +24,9 @@
     h_over_h0, luminosity_distance, distance_modulus, scale_factor_of_chi, \
     omega_x, rho_x
 
+# Boltzmann solvers
+from .boltzmann import get_camb_pk_lin, get_isitgr_pk_lin, get_class_pk_lin
```

Comment:
why did you need this?

Reply:
Why not? It might be useful for anyone who wants to get the camb or class power spectra in a `Pk2D` object in a single line of code. Plus, these functions are not prepended with an underscore, so I don't see why they shouldn't be exposed.